src/assets/i18n/de.json

```diff
@@ -119,7 +121,14 @@
     "systemSettings": "Systemeinstellungen",
     "exploreTrajectoryTitle": "Erkunde deinen Standortverlauf",
     "showNotifications": "Benachrichtigungen anzeigen",
-    "loading": "Warte…"
+    "loading": "Warte…",
+    "agreement_start": "Ich stimme der ",
```

Comment:
You're mixing camelCase and snake_case here. I think we're mostly using camelCase in these files.